Templates/AppSource App/.github/workflows/PullRequestHandler.yaml

```diff
@@ -6,6 +6,10 @@ on:
       - '**.md'
     branches: [ 'main' ]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
```

Comment:
Is `github.head_ref` unique across forks?

Reply:
concurrency is not active across forks

Reply:
I meant if `github.head_ref` identifies the head repo.

If I submit a PR from the _main_ branch of my fork, would that cancel the build on your PR, that is from the _main_ branch on your fork?

Reply:
Good question.. I'll investigate

Reply:
Updated this to use github.event.pull_request.number. Then it shouldn't matter if the branch is from the local repo or from a fork